crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -206,8 +206,15 @@ impl Format<PyFormatContext<'_>> for FormatDanglingComments<'_> {
             .iter()
             .filter(|comment| comment.is_unformatted())
         {
-            if first && comment.line_position().is_end_of_line() {
-                write!(f, [space(), space()])?;
+            if first {
+                match comment.line_position {
+                    CommentLinePosition::OwnLine => {
+                        write!(f, [hard_line_break()])?;
+                    }
+                    CommentLinePosition::EndOfLine => {
+                        write!(f, [space(), space()])?;
+                    }
+                }
             }
```

Comment:
I feel like this is the behavior we want... Otherwise, if you _only_ have own-line dangling comments, you end up with formatting like:
```python
(
    thing#dangling comment doesn't get spaced properly
    ()
)
```

This doesn't just apply to dangling comments in this position -- I've run into it elsewhere too. But if this is _intentionally_ non-breaking, I can also handle it specifically in arguments like I've done elsewhere.